R/utils_render_register_mds.R

```diff
@@ -1,23 +1,10 @@
-#' Function to load the markdown table
-#' 
-#' @param template_path The path to the markdown template
-#' @return The markdown table template
-load_md_template <- function(template_path){
-  if (!file.exists(template_path)){
-    stop("No register table template found")
-  }
-
-  md_table <- readLines(template_path)
-  return(md_table)
-}
-
 #' Function to add the markdown title based on the specific register table name.
 #' 
 #' @param markdown_table The markdown template where the title needs to be added
 #' @param register_table_name The name of the register table
 #'
 #' @return The modified markdown table
-add_markdown_title <- function(filter, md_table, register_table_name){
+add_markdown_title <- function(filter, md_table, register_table_name, filter_subcat=NULL){
   if (filter == "none"){
```

Comment:
Is there a reason here to use a string constant for "no filter", or could we also set filter to `NULL` or `NA` ?

Reply:
See comment on the next convo